pkg/package-server/apis/operators/packagemanifest_types.go

```diff
@@ -46,6 +46,9 @@ type PackageManifestStatus struct {
 	// PackageName is the name of the overall package, ala .
 	PackageName string
 
+	// Deprecation determines if the package is deprecated.
```

Comment:
nit for all the comments leading similar members: 
```suggestion
	// Deprecation is an optional message which only exists if the package is deprecated.
```